memphis/memphis.py

```diff
@@ -505,6 +505,7 @@ async def consumer(
             real_name = consumer_name.lower()
             if generate_random_suffix:
                 consumer_name = self.__generateRandomSuffix(consumer_name)
+                real_name += "_" + consumer_name.split("_")[-1]
```

Comment:
This change is wrong since real name is the consumer name without the suffix which is the desired behavior